openedx/core/lib/xblock_builtin/xblock_discussion/static/js/mathjax_include.js

```diff
@@ -0,0 +1,35 @@
+var vendorScript;
+if (typeof MathJax === 'undefined') {
+    vendorScript = document.createElement('script');
+    vendorScript.onload = function() {
+        'use strict';
+        var MathJax, setMathJaxDisplayDivSettings;
```

Comment:
This `MathJax` variable is obscuring `window.MathJax`.

Please change it to: `var MathJax = window.MathJax, setMathJaxDisplayDivSettings;`